npm/GetLastPackageVersion.ps1

```diff
@@ -0,0 +1,18 @@
+<#
+.SYNOPSIS
+This script get the last version of a given package on the npm repository
+
+.PARAMETER
+packageName : is the name of the package as referenced on npm
+#>
+
+param (
+	[Parameter(Mandatory = $true)]
+	[string]$packageName
+)
+
+$json = (npm info $packageName -json) | ConvertFrom-Json
```

Comment:
this is not actually a json, it's an object. the name of the variable is a bit misleading :)